### ISSUE TITLE
UD/UCCA Mismatches

### ISSUE THREAD

**Issue by omriabnd** [UniversalConceptualCognitiveAnnotation/UCCA_English-EWT#6]
@nschneid wrote:

Other types of mismatches I noticed:

- Scene-evoking dependent—preposition or attributive adjective/possessive: my BMW 330i, Atop Spring Mountain
- Articles attached at the wrong level in UCCA: the doctor 's office
- Relators attached at the wrong level in UCCA: distain [for] [its customers]
- PPs attached at a different level (verb vs. NP) where it's sort of ambiguous
- Multiword preposition ("out of", "off of") split in UCCA: https://github.com/UniversalConceptualCognitiveAnnotation/UCCA_English-EWT/blob/master-images/160073-0006.svg looks like an error, maybe also https://github.com/UniversalConceptualCognitiveAnnotation/UCCA_English-EWT/blob/master-images/221664-0004.svg

**Here is a view of all unmatched participants in EWT train+dev: https://docs.google.com/spreadsheets/d/1AC04Yy1NcE7l0xLst3-TjDWeKXTK57jPTb8_xkvXE24/edit#gid=1927889320&fvid=473704370
Punctuation is ignored in the matching, so that's not the reason.
I guess many cases are due to Function and Relator units,
and for the Participant Scenes it's mostly linkage mismatches.**
 
When a noun is modified by a relative clause (which seems rather common in reviews), in UCCA the relative clause will sometimes be split to a Parallel Scene and then it's a separate unit.

Is it an UCCA error to not treat a relative clause as an E-scene?
E.g. "the software I included on my resume" (https://github.com/UniversalConceptualCognitiveAnnotation/UCCA_English-EWT/blob/master-images/270502-0003.svg) has "included" as the main relation, which has to be an error

Are nonrestrictive relative clauses treated as parallel scenes in UCCA?

There are also NP coordinations which are annotated as parallel scenes: https://github.com/UniversalConceptualCognitiveAnnotation/UCCA_English-EWT/blob/master-images/225632-0008.svg looks like an error
 
** There are also many cases of copular clauses, where in UD both the copula and any modifier of the noun are dependents of the noun, but in UCCA only the NP is a Participant.**

Ah, this is huge. Because of how UD treats copulas, the subject and copula are dependents of the complement, but in UCCA usually the complement alone is a Participant.

Interestingly, "X's own" is a unit in UCCA but usually not a UD constituent (https://github.com/UniversalDependencies/docs/issues/638). I'm not actually sure what the ideal semantics is for this construction.

**Comment by dotdv** [comment]
> Are nonrestrictive relative clauses treated as parallel scenes in UCCA?

We differentiate between:

1) If the RC modifies a noun that serves as a scene element  (say A) then the RC should be included as an E-scene:
a) This is true for non-scene evoking nouns like software: [They_A tested_P me_A [on_R the_F software_C [I included on my resume (software)_A]_E]_A]]_H
b) It is also true for roles that evoke P/A or S/A like judge: [[The_F [judge_P/A]_C [who_R swims_P (judge)_A]_E]_A reached for the book.]_H
c) As for scene-evoking nouns like wedding, the guidelines are less clear about those but they are also not as frequent, I think. A few months back we told annotators that in such cases the RC should be separated into a parallel scene, but now we've been discussing that maybe it will be better to just treat them like we treat the cases above (as an E-scene): [[The_F [wedding_P]_C [I_A went_P [to_R (wedding)_P]_A]_E]_A was_F attended_P [by famous people]_A]_H

2) On the other hand where an RC modifies a noun which serves as the P or S of a top-level (H) scene then we have no other option than to separate the RC into a  parallel scene: 

- [I_A [have_F a_F friend_C]_S/A]_H who_L [(friend)_A drove_P all the way from Tel-Aviv] to_L [(friend)_A get_P here]_H. 
- a relevant guidelines example (p.28) ``[The_F wedding_P]_{H-} [I_A went_P [to_R (wedding)_P]_A]_H [was_F wonderful_D]_{-H}'' (I can think of another possible solution that was raised for this example, but either way the RC won't be E-scene. the 2nd option: [[The_F wedding_P]_{A-} I_A went_P [to_R]_{-A}])_H [(wedding)_A was_F wonderful_D]_H

**Comment by nschneid** [comment]
@dotdv Sounds good to me. Would be good to have a section explaining that in the guidelines.

> * a relevant guidelines example (p.28) ``[The_F wedding_P]_{H-} [I_A went_P [to_R (wedding)P]A]H [was_F wonderful_D]{-H}'' (I can think of another possible solution that was raised for this example, but either way the RC won't be E-scene. the 2nd option: [[The_F wedding_P]{A-} I_A went_P [to_R]{-A}])_H [(wedding)_A was_F wonderful_D]_H

I think these are almost equivalent except for which scene has a remote unit and which has a primary unit. The first one basically says "The wonderful wedding. I went to it." whereas the second says "I went to the wedding. It was wonderful." Except [(wedding)_A was_F wonderful_D]_H is missing a predicate. Should it be [(wedding)_P was_F wonderful_D]_H? Or should "wonderful" or "was" be S?

**Comment by nschneid** [comment]
Or even, "I went to the wonderful wedding"—if we're going to ignore that fact that "was wonderful" is a separate predication by making "wonderful" a D, then I don't see why we need parallel scenes: 

I_A went_P [to_R [The_F wedding_P was_F wonderful_D]_C]_A
in sentence order:
[[The_F wedding_P]\_{C-}]\_{A-} I_A went_P [to_R  was_F wonderful_D]\_{-C}]\_{-A}

**Comment by dotdv** [comment]
> > * a relevant guidelines example (p.28) ``[The_F wedding_P]_{H-} [I_A went_P [to_R (wedding)P]A]H [was_F wonderful_D]{-H}'' (I can think of another possible solution that was raised for this example, but either way the RC won't be E-scene. the 2nd option: [[The_F wedding_P]{A-} I_A went_P [to_R]{-A}])_H [(wedding)_A was_F wonderful_D]_H
> 
> I think these are almost equivalent except for which scene has a remote unit and which has a primary unit. The first one basically says "The wonderful wedding. I went to it." whereas the second says "I went to the wedding. It was wonderful." Except [(wedding)_A was_F wonderful_D]_H is missing a predicate. Should it be [(wedding)_P was_F wonderful_D]_H? Or should "wonderful" or "was" be S?

RIght, my mistake. In the 2nd option I meant to write [(wedding)_P was_F wonderful_D]_H
